src/layouts/DefaultLayout.tsx

```diff
@@ -33,6 +33,14 @@ const DefaultLayout: React.FC<Props> = ({ children, metaTag }: Props) => (
           </a>
         </Link>
       }
+      uploadLink={
+        <Link href={pathList.upload}>
+          {/* eslint-disable-next-line jsx-a11y/anchor-is-valid */}
+          <a className="button" style={{ margin: '0.5rem 0.75rem' }}>
+            <p className="has-text-black">猫ちゃん画像をアップロード</p>
```

Comment:
PCで確認した時に「他の猫ちゃんを表示」ボタンとテキストの色が違うのが気になった！あえて変えている訳ではなければ、同じ色のようが良いかも！🐱

```suggestion
            <p className="has-text-gray">猫ちゃん画像をアップロード</p>
```

Reply:
コメントありがとう！本当だね、提案してもらった案をコミットしたよ🐱！